testsuites/listener/shared/performance/conftest.py

```diff
@@ -0,0 +1,184 @@
+
+    try:
+        sync_gateway_version
+    except NameError:
+        log_info("Sync gateway version is not provided")
+        persist_cluster_config_environment_prop(cluster_config, 'sync_gateway_version', "")
+    else:
+        log_info("Running test with sync_gateway version {}".format(sync_gateway_version))
+        persist_cluster_config_environment_prop(cluster_config, 'sync_gateway_version', sync_gateway_version)
+
+    if xattrs_enabled:
+        log_info("Running test with xattrs for sync meta storage")
+        persist_cluster_config_environment_prop(cluster_config, 'xattrs_enabled', True)
+    else:
+        log_info("Using document storage for sync meta data")
+        persist_cluster_config_environment_prop(cluster_config, 'xattrs_enabled', False)
+
+    sg_config = sync_gateway_config_path_for_mode("listener_tests/listener_tests", sync_gateway_mode)
+
+    if not skip_provisioning:
+        log_info("Installing Sync Gateway + Couchbase Server + Accels ('di' only)")
+        cluster_utils = ClusterKeywords()
+        cluster_utils.provision_cluster(
+            cluster_config=cluster_config,
+            server_version=server_version,
+            sync_gateway_version=sync_gateway_version,
+            sync_gateway_config=sg_config
+        )
+
+    # Wait at the yeild until tests referencing this suite setup have run,
+    # Then execute the teardown
+    yield {
+        "liteserv": liteserv,
+        "cluster_config": cluster_config,
+        "sg_mode": sync_gateway_mode,
+        "xattrs_enabled": xattrs_enabled,
+        "device_enabled": device_enabled,
+        "liteserv_platform": liteserv_platform
+    }
+
+    log_info("Tearing down suite ...")
+    if not (device_enabled and liteserv_platform == "ios"):
+        liteserv.remove()
+
+
+# Passed to each testcase, run for each test_* method in client_sg folder
+@pytest.fixture(scope="function")
+def setup_client_syncgateway_test(request, setup_client_syncgateway_suite):
+    """Test setup fixture for client sync_gateway tests"""
+
+    log_info("Setting up client sync_gateway test ...")
+
+    liteserv = setup_client_syncgateway_suite["liteserv"]
+    cluster_config = setup_client_syncgateway_suite["cluster_config"]
+    xattrs_enabled = setup_client_syncgateway_suite["xattrs_enabled"]
+    device_enabled = setup_client_syncgateway_suite["device_enabled"]
+    liteserv_platform = setup_client_syncgateway_suite["liteserv_platform"]
+    test_name = request.node.name
+
+    if request.config.getoption("--liteserv-platform") == "macosx" and \
+            str(request.config.getoption("--liteserv-version")).startswith("1.3.1") and \
+            str(test_name).startswith("test_longpoll_changes_termination"):
+        pytest.skip("test_longpoll_changes_termination tests are known to fail on macosx with 1.3.1 CBL")
+
+    client = MobileRestClient()
+
+    # Start LiteServ and delete any databases
+    log_info("Listener going to start and launch,")
```

Comment:
"Starting LiteServ" is probably better

Reply:
done

Reply:
I still see the same log message, not sure if the change is not committed

Reply:
really! I committed and pushed . Interesting.